src/Swashbuckle.AspNetCore.SwaggerGen/DependencyInjection/SwaggerGenOptionsExtensions.cs

```diff
@@ -328,93 +328,193 @@ public static void SupportNonNullableReferenceTypes(this SwaggerGenOptions swagg
         /// <summary>
         /// Extend the Swagger Generator with "filters" that can modify Schemas after they're initially generated
         /// </summary>
-        /// <typeparam name="TFilter">A type that derives from ISchemaFilter</typeparam>
+        /// <typeparam name="TFilter">A type that derives from <see cref="ISchemaFilter"/></typeparam>
         /// <param name="swaggerGenOptions"></param>
         /// <param name="arguments">Optionally inject parameters through filter constructors</param>
         public static void SchemaFilter<TFilter>(
             this SwaggerGenOptions swaggerGenOptions,
             params object[] arguments)
             where TFilter : ISchemaFilter
         {
+            swaggerGenOptions = swaggerGenOptions ?? throw new ArgumentNullException(nameof(swaggerGenOptions));
```

Comment:
I'm not a fan of this style as in the case it's not null then it's a redundant re-assignment.